OmniBLE/PumpManagerUI/Views/OmniBLESettingsView.swift

```diff
@@ -349,45 +354,61 @@ struct OmniBLESettingsView: View  {
 
             Section() {
                 HStack {
-                    FrameworkLocalText("Pod Activated", comment: "Label for pod insertion row")
+                    FrameworkLocalText("Pod Activated", comment: "Label for Pod Activated row")
                     Spacer()
                     Text(self.viewModel.activatedAtString)
                         .foregroundColor(Color.secondary)
                 }
 
                 HStack {
-                    if let expiresAt = viewModel.expiresAt, expiresAt < Date() {
-                        FrameworkLocalText("Pod Expired", comment: "Label for pod expiration row, past tense")
-                    } else {
-                        FrameworkLocalText("Pod Expires", comment: "Label for pod expiration row")
-                    }
+                    FrameworkLocalText("Pod Expiration", comment: "Label for Pod Expiration row")
                     Spacer()
                     Text(self.viewModel.expiresAtString)
                         .foregroundColor(Color.secondary)
                 }
 
+                HStack {
+                    if viewModel.podDetails?.fault != nil {
+                        FrameworkLocalText("Pod Faulted", comment: "Label for Pod Faulted row")
+                        Spacer()
+                        Text(self.viewModel.faultedAtString)
+                            .foregroundColor(Color.secondary)
+                    } else if viewModel.podOk {
```

Comment:
Delete the "if viewModel.podOk " portion of this line to prevent a blank row display for this row when there is no pod to match the behavior with no pod of all the other rows in this section